docs/CONFIG.md

```diff
@@ -7209,6 +7386,17 @@ block, but it is possible to receive a head BEFORE that block is actually
 available from the connected node via RPC, due to race conditions in the code of the remote ETH node. In this case you will get false
 "zero" blocks that are missing transactions.
 
+### FinalizedBlockOffset
```

Comment:
I find it difficult to understand the usage of this config, and I'm worried that someone without context will be able to understand. A few comments on this. First I think we should explain here what this config actually accomplishes, since here we only explain what it does. Second the CAUTION message below doesn't make it clear to me why transaction production will get delayed. I have an idea, but again someone without context won't be able to understand. Third, and less important, I'm worried about the introduction of this config in a sense that FinalityDepth doesn't have the same meaning anymore across the codebase.

Reply:
Good point. Updated the docs